client/src/views/client-area/statistics/CustomOrderStatisticsPage.vue

```diff
@@ -0,0 +1,666 @@
+            </div>
+          </ion-card>
+          <ion-card class="users-cards-list-item">
+            <div class="users-cards-list-item-icons">
+              <ion-icon
+                class="users-cards-list-item-icons__main"
+                :icon="person"
+              />
+              <ion-icon
+                class="users-cards-list-item-icons__secondary"
+                :icon="arrowForward"
+              />
+            </div>
+            <div class="users-cards-list-item-text">
+              <ion-text class="users-cards-list-item-text__number title-h1">
+                {{ organizationStats.outsiderUsersDetail.revoked }}
+              </ion-text>
+              <ion-text class="users-cards-list-item-text__text subtitles-sm">
+                {{ $msTranslate({ key: 'clientArea.statistics.outsider', count: organizationStats.outsiderUsersDetail.revoked }) }}
+              </ion-text>
+            </div>
+          </ion-card>
+        </div>
+      </div>
+
+      <!-- storage -->
+      <div class="storage">
+        <ion-text class="storage__title title-h2">{{ $msTranslate('clientArea.statistics.titles.storage') }}</ion-text>
+        <div class="storage-data">
+          <div class="storage-data-global">
+            <ion-text
+              class="storage-data-global__total title-h1"
+              :class="{ 'overused': storagePercentage >= 100}"
+            >
+              {{ $msTranslate(formatFileSize(totalData)) }}
+            </ion-text>
+            <div class="storage-data-global-info">
+              <ion-text class="storage-data-global-info__text body">{{ $msTranslate('clientArea.statistics.ofWhich') }}</ion-text>
+              <ion-text class="storage-data-global-info__text">
+                <span class="title-h5">{{ $msTranslate(formatFileSize(organizationStats.dataSize)) }}</span>
+                <span class="body">{{ $msTranslate('clientArea.statistics.data') }}</span>
+              </ion-text>
+              <ion-text class="storage-data-global-info__text">
+                <span class="title-h5">{{ $msTranslate(formatFileSize(organizationStats.metadataSize)) }}</span>
+                <span class="body">{{ $msTranslate('clientArea.statistics.metadata') }}</span>
+              </ion-text>
+            </div>
+          </div>
+          <div class="storage-data-consumption">
+            <ion-text class="storage-data-consumption__title title-h5">
+              {{ $msTranslate('clientArea.statistics.consumptionDetail') }}
+            </ion-text>
+
+            <div class="consumption">
+              <ms-report-text
+                :theme="MsReportTheme.Warning"
+                v-if="storagePercentage === 100"
+              >
+                {{ $msTranslate('clientArea.statistics.fullUsed') }}
+              </ms-report-text>
+              <ms-report-text
+                :theme="MsReportTheme.Error"
+                v-if="storagePercentage > 90 && storagePercentage < 100"
```

Comment:
You could just write `v-else-if="storagePercentage > 90"`. The case where it's 100 would be covered by the first `v-if`.